### DISCUSSION TITLE
Deepgram base URL Whitelist

### DISCUSSION THREAD

**Discussion by tsaqueton** [deepgram/community#575] [General help] [answered]
I am writing a program for a company with high security that wants to whitelist all external connections/API calls. What would be the base URL for Deepgram that they could whitelist to allow all Deepgram API calls?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@tsaqueton  you can refer to our [API References ](https://developers.deepgram.com/reference/deepgram-api-overview)for all the base URLS you'd want to consider white listing depending on what you are attempting to do with Deepgram.
